src/components/FeaturePage/feature-page.module.css

```
@@ -0,0 +1,290 @@
+/* ─────────────────────────────────────────────────────────────
+   FeaturePage — styles (extends paper theme)
+   ───────────────────────────────────────────────────────────── */
+
+/* Compose shared tokens */
+.section     { composes: section     from '../PaperLayout/paper.module.css'; }
+.sectionAlt  { composes: sectionAlt  from '../PaperLayout/paper.module.css'; }
```

Stylelint (17.12.0)

[error] Expected "sectionAlt" to be "sectionalt" (value-keyword-case)

(value-keyword-case)